src/components/CollaboratorModal/submodals/MainSubmodal.tsx

```diff
@@ -119,20 +141,17 @@ const MainSubmodal = () => {
           isDisabled={collaboratorRoleData?.role !== "ADMIN"}
         >
           <FormLabel>Only admins can add or remove collaborators</FormLabel>
-          <Input
-            onChange={(event) => {
-              setNewCollaboratorEmail(event.target.value)
-              setAddCollaboratorError("")
-            }}
-          />
-          <FormErrorMessage>{addCollaboratorError}</FormErrorMessage>
+          <Input {...register("newCollaboratorEmail")} />
```

Comment:
Should we also add a validation here for getting a valid email address? Right now this functionality is entirely offloaded to the backend but was thinking it might be better to get immediate feedback if the user entered an invalid email address.

Reply:
yea, agree - see above!